hathor/conf/testnet.yml

```diff
@@ -37,34 +37,4 @@ CHECKPOINTS:
   1_600_000: 00000000060adfdfd7d488d4d510b5779cf35a3c50df7bcff941fbb6957be4d2
 
 FEATURE_ACTIVATION:
-  evaluation_interval: 40_320
```

Comment:
I guess it's missing the `evaluation_internal: 20160`, isn't it?

Reply:
Nope, `20160` is the default value in the configuration class, so I'm just removing an override. For mainnet it's the same, this key is not being set. Now both mainnet and testnet don't need it.

Reply:
Got it. Thanks!